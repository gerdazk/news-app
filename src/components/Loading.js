import React from "react";

function Loading() {
  return <div className="loading" data-testid="loading"></div>;
}

export default Loading;
