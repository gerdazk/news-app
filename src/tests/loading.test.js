import Loading from "../components/Loading";
import React from "react";
import { mount } from "enzyme";

test("Loading components matches snapshot:", () => {
  const wrapper = mount(<Loading />);
  expect(wrapper).toMatchSnapshot();
});
