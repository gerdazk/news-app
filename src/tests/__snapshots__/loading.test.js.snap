// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading components matches snapshot: 1`] = `ReactWrapper {}`;
